OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -307,6 +307,12 @@ bool IsBlockedBy(Actor self, Actor otherActor, Actor ignoreActor, CellConditions
 			// If we have a real actor, we can then perform the extra checks that allow us to avoid being blocked.
 			if (self == null)
 				return true;
+
+			// If self is structure producing units we are not blocked

```

Comment:
This doesn't make sense: if `self` is a structure, then this code is guaranteed to never run (`Mobile` is for ground based units)

Reply:
I know it doesn't make sense but for some reason this exact thing kept happening with airstrip in cnc resulting in crash (null reference exception).

While the exception itself was thrown in https://github.com/GSonderling/OpenRA/blob/83f2cc008a4129fd6f4feec877b8332ebaffcde2/OpenRA.Mods.Common/Traits/Production.cs#L69

the problems start in Mobile.cs. Usually, but not always, the self was airstrip and otherActor was bike or light tank. 

It happened with other units too but these were most frequent.